### PR TITLE
Saved location API request

### PR DESCRIPTION
**Pull request by codemutatio**
Validate, Create, get, Update and delete saved Location

### CODE REVIEW

**Comment by Simpcyclassy** [comment]
Looks good to me

File: functions/src/validations/savedLocationInput.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const _ = require('lodash');
+
+const arrayCompare = (expectedData, inputData) => {
+    const stringExpectedData = expectedData.sort();
+    const stringInputData = inputData.sort();
+    return JSON.stringify(stringExpectedData) === JSON.stringify(stringInputData);
+};
+const validateSavedLocationInput = datas => {
+    const errors = {};
+    const expectedData = ['name', 'price', 'trafficRate', 'address', 'state', 'lga', 'country'];
```
Comment (enyeInc) [inline functions/src/validations/savedLocationInput.js]: move this to constants file

File: functions/src/validations/savedLocationInput.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const _ = require('lodash');
+
+const arrayCompare = (expectedData, inputData) => {
```
Comment (enyeInc) [inline functions/src/validations/savedLocationInput.js:11]: your functions need comments

**Review by enyeInc — APPROVED** [review]
Code looks clean, a couple minor things to fix but LGTM

**Pull request merged**